lib/components/primitive-components/Group/Group.ts

```diff
@@ -308,18 +308,48 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
     }
   }
 
+  _computeSchematicPositionBeforeLayout(): void {
+    const { db } = this.root!
```

Comment:
This method shouldnt mutate, never mutate db unless you're definitely inside a render phase and the name of the method makes sense to mutate